data/styles/clear/include/Basemap.mapcss

```diff
@@ -670,34 +672,48 @@ area|z18-[building=train_station],
 /* 8.3 Barrier */
 
 line|z16-[barrier=wall],
-line|z16-[barrier=city_wall],
 line|z16-[barrier=retaining_wall],
 line|z16-[barrier=hedge],
-line|z16-[barrier=fence],
 line|z16-[historic=citywalls],
```

Comment:
I think it makes sense to render all types of walls similarly?
And make only fences and hedges different.

Reply:
sure, will update the PR